lib/pages/clouds-page.tsx

```diff
@@ -0,0 +1,556 @@
+/* 2021-05-31 Dave Weaver, @webaissance - based on waves-page.tsx */
+/* IMPORTS */
+import React, { useState } from "react";
+import { Page } from "../page";
+import {
+  CompositionContextWidget,
+  createSvgCompositionContext,
+} from "../svg-composition-context";
+import { Checkbox } from "../checkbox";
+import { RandomizerWidget } from "../randomizer-widget";
+/* TO DO: Get symbols randomly or from a pulldown */
+/* CLOUD SVG */
+const Cloud: React.FC<{
+  stroke: string;
+  fill: string;
+	strokewidth: number; 
+}> = ({ stroke, fill, strokewidth}) => (
+  <>
+<path fill={fill} fillRule="evenodd" stroke="none" d="M 360.000 596.177 C 500.811 596.177 614.404 482.584 614.404 341.774 C 614.404 200.963 500.811 87.370 360.000 87.370 C 219.189 87.370 105.596 200.963 105.596 341.774 C 105.596 482.584 219.189 596.177 360.000 596.177 Z"/>
+<path fill="none" stroke={stroke} strokeWidth={strokewidth} strokeLinecap="round" strokeLinejoin="round" d="M 604.098 371.149 C 602.507 412.416 597.237 433.275 575.983 476.189 M 530.256 180.457 C 583.769 228.655 607.192 290.877 604.098 371.149 M 212.437 216.472 C 280.858 106.327 437.914 97.287 530.256 180.457 M 208.196 406.565 C 175.078 346.560 176.784 273.866 212.437 216.472 M 437.019 451.501 C 365.701 513.810 255.178 491.691 208.196 406.565 M 479.177 353.518 C 480.210 390.509 464.892 427.148 437.019 451.501 M 383.232 252.404 C 437.329 252.139 477.731 301.781 479.177 353.518 M 315.724 293.646 C 329.002 268.221 355.037 252.543 383.232 252.404 M 324.498 355.831 C 305.359 339.494 306.332 311.630 315.724 293.646 M 347.839 363.173 C 337.678 362.940 332.207 362.412 324.498 355.831 M 366.134 334.409 C 377.849 347.389 363.196 363.525 347.839 363.173 M 354.460 336.520 C 354.806 332.878 362.306 330.168 366.134 334.409 M 105.596 341.774 C 105.596 482.584 219.189 596.177 360.000 596.177 C 500.811 596.177 614.404 482.584 614.404 341.774 C 614.404 200.963 500.811 87.370 360.000 87.370 C 219.189 87.370 105.596 200.963 105.596 341.774 "/>
+  </>
+);
+
+
+/* INITIAL VALUES */
+const CLOUD_STROKE = "#79beda";
+const CLOUD_FILL = "#2b7c9e";
+const CLOUD_STROKEWIDTH = 30;
+const NUM_ELEMENTS = 7;
+const NUM_CLOUDS = 12;
+//const MAX_CLOUDS = 12;
+const CLOUD_DISTANCE = 9;
+const NUM_ROWS = 3;
+const ROW_DISTANCE = 5;
+const CLOUD_SPEED = 3;
+const PARALLAX_SPEED = 2;
+const PARALLAX_SIZE = 1;
+const ROTATION_SPEED = 6;
+const CLOUD_SPACING = 0.5;
+const CLOUD_SCALE = 0.8;
+const CLOUD_PULSEMIN = 0.8;
+const CLOUD_PULSE = 1.7;
+const PULSE_DURATION = 2.6;
+const BG_COLOR = "#FFFFFF";
+const SCREEN_WIDTH = 12000;
+const MASK_RADIUS = 2000;
+const MASK_X = 4500;
+const MASK_Y = 3000;
+
+const NumericSlider: React.FC<{
```

Comment:
Is there a reason you've "forked" the `NumericSlider` component here instead of reusing the one in `numeric-slider.tsx`?  The latter uses the correct CSS classes, which ensures that the layout is correct (see my earlier comment for a screenshot), and we would like to keep things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)...

Reply:
@toolness - thanks for your comments! When I first started the Clouds page I just copied the Waves page. At that time the Numeric Slider definition was built in to the Waves page - so therefore it was in the Clouds page as well. I now see you've modernized the Waves page - and the Numeric Slider is being included - so I'll do that on the Clouds page as well. Thanks for pointing that out!

Reply:
Ahhh gotcha, my bad!